pype/plugins/blender/load/load_model.py

```diff
@@ -7,20 +7,16 @@
 
 from avalon import api, blender
 import bpy
-import pype.hosts.blender.plugin
+import pype.hosts.blender.plugin as plugin
 
 logger = logging.getLogger("pype").getChild("blender").getChild("load_model")
```

Comment:
Advice for future PRs: All avalon plugins has attribute `log` you don't have to create new logger, just use `self.log.*` in the loader.